tools/spelling/spelling_dictionary.txt

```diff
@@ -231,6 +231,7 @@ PostCBs
 PREBIND
 PRNG
 PROT
+PSS
```

Comment:
If appropriate, these can be contained in backticks in comments and the spellchecker will ignore them, and then they don't need to be added to the dictionary.